WeakAuras/WeakAuras.lua

```diff
@@ -3230,35 +3230,40 @@ local glow_frame_monitor
 local anchor_unitframe_monitor
 Private.dyngroup_unitframe_monitor = {}
 do
-  local function frame_monitor_callback(event, frame, unit)
```

Comment:
In LibGetFrame, you can optimize CacheMonitorMixin:Add, by checking
if type(self.data[key]) ~= "table" and f self.data[key] == nil
before iterating. You basically only need to iterate over all args if type(self.data[key]) == table.

Reply:
Implemented in https://github.com/mrbuds/LibGetFrame/commit/8abfd86e2038ae42ecc05f9082e6a778eaa41f30